### PR TITLE
fix: making table with ticket details compatible with phone

### LINKED ISSUE #3511: The table with ticket details is not responsive in phone.
**Is your feature request related to a problem? Please describe.**
The table with ticket details on the events page is not responsive in mobile phone view and there is an overflow on the x-axis of the entire page on the phone due to same reason.

**Describe the solution you'd like**

I would like to use bootstrap or Bulma to make the tables more responsive and better looking.

**Describe alternatives you've considered**

**Additional context**

issue1

Please assign me the task to implement this. Thank you.

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3511
### issue it resolves
ticket detail table was not showing properly .
#### Changes proposed in this pull request:
- making table with ticket details responsive with phone.
- donation ticket is showing properly in mobile also .
- making segment compact and table small for mobile to view properly .
Screenshot from 2019-10-30 17-14-01
Screenshot from 2019-10-30 17-14-06

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/public/ticket-list.hbs [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {{!-- Removed `Currency Symbol` helper from the `paymentCurrency` since some currency are not getting rendered. Hotfix until a permanent solution found --}}
 <form class="ui form" {{action on='submit' preventDefault=true}}>
-  <table class="ui unstackable very basic compact table" style="margin-bottom: 0">
+  <table class="ui unstackable {{if device.isMobile 'small'}} very basic compact table" style="margin-bottom: 0">
```
Comment (kushthedude) [inline app/templates/components/public/ticket-list.hbs]: Screenshot 2019-10-30 at 7 25 53 PM

Solve CSS Overflow
--

File: app/templates/components/public/ticket-list.hbs [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
     <thead class="full-width">
       <tr>
         <th>{{t 'Type'}}</th>
         <th class="four wide">{{t 'Sales Ends'}}</th>
-        <th class="ui three wide">{{t 'Price'}}</th>
+        <th class="ui three wide single line">{{#if device.isMobile}}<i class="money bill alternate outline icon"></i>{{/if}}{{t 'Price'}}</th>
```
[the remark this answers is not in the document]
Comment (maze-runnar) [reply app/templates/components/public/ticket-list.hbs]: I added icon only in mobile view to give price column some more width to show donation ticket price input box correctly .

[the remark this answers is not in the document]
Comment (maze-runnar) [reply app/templates/components/public/ticket-list.hbs]: I tried many ways but the table is under a segment so nothing is working for giving more width to that price column . that icon remove the requirement of an extra div may be. :slightly_smiling_face:

File: app/templates/components/public/ticket-list.hbs (hunk 1 above)
Comment (maze-runnar) [reply app/templates/components/public/ticket-list.hbs]: sir it due to "ui segment"  class and it is also exist before my implementation in actual website.
Screenshot from 2019-10-30 20-16-02
It is visible fine in galaxy5, and pixel2 devices, but showing this overflow in very small devices .

Comment (maze-runnar) [reply app/templates/components/public/ticket-list.hbs]: please help me with this issue if there is any thing else

File: app/templates/components/public/ticket-list.hbs (hunk 2 above)
[the remark this answers is not in the document]
Comment (maze-runnar) [reply app/templates/components/public/ticket-list.hbs]: > It looks odd

please suggest me anything else.

File: app/templates/components/public/ticket-list.hbs (hunk 1 above)
Comment (kushthedude) [reply app/templates/components/public/ticket-list.hbs]: I will pull and check on local tonight.

Comment (maze-runnar) [reply app/templates/components/public/ticket-list.hbs]: ok

**Comment by snitin315** [comment]
change ~~`fix :`~~  `fix:` and update your branch as well.

**Comment by maze-runnar** [comment]
x-scrollable is not making any difference -
Screenshot from 2019-11-18 20-45-40
"Donation ticket" price is not visible correctly .

**Comment by kushthedude** [comment]
Please try to improve the responsiveness avoding `device.*` property, DOM Modifications are way costlier then CSS queries

**Comment by maze-runnar** [comment]
working fine now with iphone5 too .:
Screenshot from 2019-11-24 14-29-55
Screenshot from 2019-11-24 14-29-58

**Comment by maze-runnar** [comment]
proper space for donation tickets , and also remove {{if device.isMobile}} and icon .
Screenshot from 2019-11-24 15-12-53
Screenshot from 2019-11-24 15-17-47

**Comment by maze-runnar** [comment]
> Please try to improve the responsiveness avoding `device.*` property, DOM Modifications are way costlier then CSS queries

Done :smile:  and thanks

**Comment by kushthedude** [comment]
> CSS

@maze-runnar Can you please update the screenshots and resolve the conflicts for the PR ?

**Pull request merged**